bin/docserv-createconfig

```diff
@@ -194,26 +200,73 @@ def switchbranch(repo):
         # change back to initial branch
         repo.checkout(initial_ref)
 
-def get_dc_files(path, branches, filter_dc=None):
-    """Collect all DC files in branches.
+def get_deliverables(path, branches, filter_dc=None):
+    """Collect all DC files and subdeliverables in all branches.
 
     :param str path: the path to the Git repository
     :param dict branches: A dictionary with lists as keys like
          {'branch1': [], 'branch2': [], ...}
     :param filter_dc: filter
-    :return: a dictionary with all DC files found in this branch
-    :rtype: dict
+    :return: a tuple with two dictionaries for each branch:
+        first dictionary contains all DC files
+        second dictionary contains all subdeliverables
```

Comment:
If we ever need to expand this, e.g. to cover the case of `DC-*-html` instead of `DC-*-all` or the case that there are multiple sets like in SOC, maybe you could update the data structure so that we can associate each subdeliverable with the DC file it needs to go into.

E.g. something more like:

```
{ 'branch1': {
    'DC-blub-all' : ['book.bla', 'book.blu'],
    'DC-blub-bla' : [],
    'DC-blub-blu' : [], ... }, 
  'branch2': {
    'DC-blubs-all' : ['book.ble', 'book.bli'],
    'DC-blubs-ble' : [],
    'DC-blubs-bli' : [], ... }
}
```

I know this will probably create the necessity to change a few more functions ... sorry.

(Feel free to use data types as appropriate of course, I am not the firmest in that area of expertise.)

Reply:
Thanks, I've already pushed some changes :)